project/Doc.scala

```diff
@@ -50,7 +51,7 @@ object Scaladoc extends AutoPlugin {
       docs
     })
 
-  def scaladocOptions(ver: String, base: File, plugins: Seq[File]): List[String] = {
+  def scaladocOptions(scalaBinaryVersion: String, ver: String, base: File, plugins: Seq[File]): List[String] = {
     val urlString = GitHub.url(ver) + "/€{FILE_PATH_EXT}#L€{FILE_LINE}"
```

Comment:
This needs to be reverted as well for pre-Scala 2.12.9.

Reply:
Ah, indeed, will do.

Does it work with 2.13.0 and was backported to 2.12.9?

Reply:
Exactly. It is 2.12.9+

Reply:
:+1:

Reply:
Great, thanks.